scenes/stealth_level/stealth_level.tscn

```diff
@@ -152,6 +154,9 @@ dialogue = ExtResource("6_oyx4c")
 
 [node name="Talker2" parent="NPCs" instance=ExtResource("5_xtu6t")]
 position = Vector2(7249, 934)
+script = ExtResource("6_1b8xw")
+quest_scene = ExtResource("7_idy4y")
+quest_description = ""
```

Comment:
This is kind of weird because the NPC here is not a quest *starter* but an ender. I guess I should make this a teleporter like in the music map.

Reply:
I agree, but I think I can fix that in the collectibles branch since this npc acting as a teleporter is temporary